influunt-app/app/scripts/controllers/planos.js

```diff
@@ -17,10 +17,13 @@ angular.module('influuntApp')
               influuntAlert, influuntBlockui, geraDadosDiagramaIntervalo,
               handleValidations) {
 
+      var LIMITE_PLANOS = 16;
```

Comment:
Ele pode ir pro model `ModeloControlador` junto com as outras configurações, se esse valor for modificável, ou então ir pro `RangeUtils` se for fixo.